core/src/main/java/com/rudderstack/android/sdk/core/util/Utils.java

```diff
@@ -60,6 +60,12 @@ public static String getTimeStamp() {
         return formatter.format(new Date());
     }
 
+    public static Long getCurrentTimeInSecondsLong() {
+        long currentTimeInMilliSeconds = System.currentTimeMillis();
+        long currentTimeInSeconds = TimeUnit.MILLISECONDS.toSeconds(currentTimeInMilliSeconds);
```

Comment:
You can use `System.currentTimeMillis()` directly here instead of taking it in a variable.

Reply:
Yeah, wrote it this way cause it seemed more readable. But I can change it, do you want me to?

Reply:
Addressed